### PR TITLE
modify the log4j config

### PR DESCRIPTION
**Pull request by niloofartrg**

### CODE REVIEW

**Comment by monperrus** [comment]
good progress, thanks. the build log is still of text: https://travis-ci.org/SpoonLabs/coming/builds/563619372

File: src/main/java/fr/inria/coming/changeminer/analyzer/instancedetector/PatternInstancesFromDiff.java [hunk 1]
```diff
@@ -57,10 +64,12 @@ public String toString() {
 			for (ChangePatternInstance instance : instances) {
 				resultString += "\n" + instance.toString();
 			}
-			return resultString += "\n----";
+            log.info(resultString += "\n----");
```
Comment (monperrus) [inline src/main/java/fr/inria/coming/changeminer/analyzer/instancedetector/PatternInstancesFromDiff.java]: undo this. a toString remains as is, we must remove the call to tostring instead.

File: src/main/java/fr/inria/coming/changeminer/analyzer/instancedetector/PatternInstancesFromDiff.java [hunk 2]
```diff
@@ -57,10 +64,12 @@ public String toString() {
...
+//			return resultString += "\n----";
 		} catch (Exception e) {
 			e.printStackTrace();
 		}
+		log.info("--Diff ex--");
```
Comment (monperrus) [inline src/main/java/fr/inria/coming/changeminer/analyzer/instancedetector/PatternInstancesFromDiff.java]: remove this

File: src/main/java/fr/inria/coming/changeminer/analyzer/instancedetector/PatternInstancesFromRevision.java [hunk 3]
```diff
@@ -35,15 +35,15 @@ public void setInstances(List<PatternInstancesFromDiff> instances) {
...
-	public String toString() {
-		// return "PatternInstancesFromRevision [instances=" + instances + "]";
-		String r = "";
-		for (PatternInstancesFromDiff patternInstancesFromDiff : infoPerDiff) {
-			r += "\n" + patternInstancesFromDiff.toString();
-		}
-
-		return r;
-	}
+//	@Override
```
Comment (monperrus) [inline src/main/java/fr/inria/coming/changeminer/analyzer/instancedetector/PatternInstancesFromRevision.java]: undo this. a toString remains as is, we must remove the call to tostring instead.

**Comment by monperrus** [comment]
you're making progress, keep up the work!

File: src/main/java/fr/inria/coming/changeminer/analyzer/instancedetector/PatternInstancesFromDiff.java (hunk 1 above)
Comment (monperrus) [reply src/main/java/fr/inria/coming/changeminer/analyzer/instancedetector/PatternInstancesFromDiff.java]: ping

**Comment by niloofartrg** [comment]
changes are done @monperrus

**Comment by monperrus** [comment]
Thanks for the progress. Merged. Now the build log fits in 24000 lines.

We probably want to remove some other `System.out` in a subsequent PR.

**Pull request merged**